Kingsley_ansible_playbook.yml

```diff
@@ -94,4 +83,17 @@
         name: todoapp
         daemon_reload: yes
         state: restarted
-      become: yes
\ No newline at end of file
+      become: yes
+
+    - name: Clone Git repository
+      ansible.builtin.git:
+        repo: 'https://github.com/kingbrookey/DevOps-Course-Starter.git'
+        dest: /opt/todoapp
+        version: module_3
+      become: yes
+      become_user: root
+     
+    - name: Install project dependencies using Poetry
+      ansible.builtin.command: /opt/todoapp/pyproject.toml install
+      become_user: ec2-user
```

Comment:
I'd expect this to run a `poetry install` rather than using the `pyproject.toml` file directly